source/MRMesh/MRObjectVoxels.cpp

```diff
@@ -575,8 +580,6 @@ void ObjectVoxels::serializeFields_( Json::Value& root ) const
     serializeToJson( vdbVolume_.voxelSize, root["VoxelSize"] );
 
     serializeToJson( vdbVolume_.dims, root["Dimensions"] );
-    serializeToJson( activeBox_.min, root["MinCorner"] );
-    serializeToJson( activeBox_.max, root["MaxCorner"] );
```

Comment:
Should we keep serializing it for compatibility with older versions of MeshInspector?